stdlib/3/os/__init__.pyi

```diff
@@ -341,3 +341,16 @@ def waitid(idtype: int, id: int, options: int) -> waitresult: ...
 P_ALL = 0
 WEXITED = 0
 WNOWAIT = 0
+
+if sys.version_info >= (3, 3):
+    def sync() -> None: ...  # Unix only
+
+    if sys.version_info >= (3, 5):
```

Comment:
Since there's no difference in signatures I'd avoid the additional version_info check here and instead add a slightly longer clarifying comment.